### PR TITLE
Issue#162 rename project and scenario

### PR DESCRIPTION
**Pull request by kamil0301**
Added renaming of project and scenario - issue #162 
Scenario renaming currently does not rename folders with comparison charts. I am working on it and will add it to this PR later.

### CODE REVIEW

**Comment by kamil0301** [comment]
As of now, comparison charts in _comp folder in a project are created each time the user want to compare scenarios, so not renaming the folders should not break anything. This should be fixed in my opinion, based on this I added the issue above

File: gui/projects.php [hunk 1]
```diff
@@ -26,13 +32,33 @@ function projects_list(){/*{{{*/
...
+	<div>Rename current scenario (<i>$current_project/$current_scenario</i>) as:
+	<form method=POST>
+		<input autocomplete=off type=text placeholder='new scenario name' name=rename_scenario required pattern='\w{1,15}' title='max 15 of alphanumeric characters'> 
+		<input autocomplete=off type=submit name=submit value='rename'>
+	</form></div>
+	";
+
+	//copy scenario
 	echo "<br>
 	<div>Copy current scenario as:
```
Comment (kowalskiw) [inline gui/projects.php:63]: Optionally we can alter also this line just to unify input fields labels:
`<div>Copy current scenario (<i>$current_project/$current_scenario</i>) as:`

**Review by kowalskiw — APPROVED** [review]
All features works fine - to be squashed&merged. @kamil0301, please consider my comment before.

File: gui/projects.php (hunk 1 above)
Comment (kamil0301) [reply gui/projects.php]: labels unified

**Pull request merged**